File: 14-longest-common-prefix/longest-common-prefix.cs
public class Solution {
    public string LongestCommonPrefix(string[] strs) {
        if (strs == null || strs.Length == 0){
            return "";
        }

        Array.Sort(strs);
        string firstWord = strs[0];
        string lastWord = strs[strs.Length - 1];
        int i = 0;

        while (i < firstWord.Length && i < lastWord.Length && firstWord[i] == lastWord[i]){
            i ++;
        }

        return firstWord.Substring(0,i);
    }
}

File: 14-longest-common-prefix/longest-common-prefix.py
class Solution:
    def longestCommonPrefix(self, strs: List[str]) -> str:
        if not strs:
            return ""
        
        strs.sort()
        first_word = strs[0]
        last_word = strs[-1]
        i = 0

        while i < len(first_word) and i < len(last_word) and first_word[i] == last_word[i]:
            i += 1

        return first_word[:i]